unit_testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -12,3 +16,19 @@ def test_sct_label_vertebrae_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+
+def test_sct_label_vertebrae_disc():
+    subprocess.check_output("sct_label_vertebrae -i sct_testing_data/t2/t2.nii.gz "
+                            "-s sct_testing_data/t2/t2_seg-manual.nii.gz "
+                            "-initfile sct_testing_data/t2/init_label_vertebrae.txt -c t2 -ofolder label_test",
+                            shell=True)
+    nifti_label = nib.load("label_test/t2_labels-disc.nii.gz")
+    image_label = np.array(nifti_label.dataobj)
+    IS_coordinates = np.nonzero(image_label)[1]  # image is AIL
+    IS_coordinates.sort()
+    real_IS_coordinates = [2, 17, 34, 48]
+    assert np.allclose(real_IS_coordinates, IS_coordinates)
```

Comment:
Is ther anything else we might be able to test here? AFAIK, `sct_label_vertebrae` generates a few different files (warping fields, straight_ref, manual_labeled...) Maybe we could check for the existence of those files? (Or, verify their contents somehow?)

Reply:
Yep yep, I wanted to starts the discussion as to how to test this. I guess we can check for the existance of teh wrapping field and straight_ref, for non labeled image it seems complicated to look into it (beside there is already a test for the straightening). We could looki into all the "labeled" output.

Reply:
Started in 3d3655d